examples/interpreter/physl.cpp

```diff
@@ -308,8 +309,12 @@ phylanx::execution_tree::compiler::result_type compile_and_run(
     phylanx::execution_tree::compiler::environment env =
         phylanx::execution_tree::compiler::default_environment();
 
+    phylanx::execution_tree::compiler::primitive_name_parts argv;
+    argv.instance = "sys_argv";
```

Comment:
Shouldn't this be `argv.primitive`? Can't you use the constructor you created?

Reply:
Why should it be `.primitive`? The primitive type is going to be define-variable.